### ISSUE TITLE
Many "is not a function" errors after first scroll, after which scrolling breaks.

### ISSUE THREAD

**Issue by bradley** [xiel/embla-carousel-wheel-gestures#36]
Not sure what's happening, because this used to work, but Im suddenly getting a lot of console errors after the first "swipe" followed by scrolling breaking all together. Any ideas?

```
Uncaught TypeError: engine.scrollBody.useDefaultSpeed is not a function
    at wheelGestureEnded (commons.js:13946)
    at handleWheel (commons.js:13990)
    at commons.js:83454
    at Array.forEach (<anonymous>)
    at dispatch (commons.js:83453)
    at publishWheel (commons.js:83622)
    at end (commons.js:83846)
4commons.js:13946 Uncaught TypeError: engine.scrollBody.useDefaultSpeed is not a function
    at wheelGestureEnded (commons.js:13946)
    at handleWheel (commons.js:13990)
    at commons.js:83454
    at Array.forEach (<anonymous>)
    at dispatch (commons.js:83453)
    at publishWheel (commons.js:83622)
    at processWheelEventData (commons.js:83689)
    at HTMLDivElement.feedWheel (commons.js:83584)
commons.js:13946 Uncaught TypeError: engine.scrollBody.useDefaultSpeed is not a function
    at wheelGestureEnded (commons.js:13946)
    at handleWheel (commons.js:13990)
    at commons.js:83454
    at Array.forEach (<anonymous>)
    at dispatch (commons.js:83453)
    at publishWheel (commons.js:83622)
    at end (commons.js:83846)
3commons.js:13946 Uncaught TypeError: engine.scrollBody.useDefaultSpeed is not a function
    at wheelGestureEnded (commons.js:13946)
    at handleWheel (commons.js:13990)
    at commons.js:83454
    at Array.forEach (<anonymous>)
    at dispatch (commons.js:83453)
    at publishWheel (commons.js:83622)
    at processWheelEventData (commons.js:83689)
    at HTMLDivElement.feedWheel (commons.js:83584)
```

**Comment by bradley** [comment]
Hmm looks like embla team may have broken this: https://github.com/davidcetinkaya/embla-carousel/commit/4f38e7ee91d45718dfac170f6a40869a132ea3df#diff-bbb406ed0487e957fb0e631fc711042557f382619e305e48761bf38ecd625b8e

**Comment by bradley** [comment]
I fixed this by reverting to version 4.0.3 of embla, but you may want to update this package so Im going to leave this open.

**Comment by davidcetinkaya** [comment]
@xiel and @bradley, I think this is caused by the internal `useDefaultSpeed` method being renamed to `useBaseSpeed` in version 4.0.6 of embla. This package needs to be updated accordingly.

**Comment by xiel** [comment]
@bradley thanks for the notice, will update this later tonight.

**Comment by davidcetinkaya** [comment]
@xiel here's is the [commit](https://github.com/davidcetinkaya/embla-carousel/commit/4f38e7ee91d45718dfac170f6a40869a132ea3df) for reference. Thanks!

**Comment by davidcetinkaya** [comment]
@xiel I can change the name back to `useDefaultSpeed` again if this is causing you a lot of headache?

Let me know if you think this is a better approach.
David

**Comment by xiel** [comment]
@davidcetinkaya the new name is fine for me :)

[fixed](https://github.com/xiel/embla-carousel-wheel-gestures/compare/v1.0.4...v1.0.6#diff-1d74079c2ba2b8a6a6831ca8eca614bc69c7aedd91e51d3e9f56b5dff4186e89L47-R52) and released with [v1.0.6](https://www.npmjs.com/package/embla-carousel-wheel-gestures/v/1.0.6)

**Comment by davidcetinkaya** [comment]
Allright then @xiel, great work 🎉 !
